src/main/java/com/googlesource/gerrit/plugins/oauth/GoogleOAuthService.java

```diff
@@ -212,9 +218,11 @@ public OAuthToken getAccessToken(OAuthVerifier rv) {
   public String getAuthorizationUrl() {
     String url = service.getAuthorizationUrl(null);
     try {
-      if (!Strings.isNullOrEmpty(domain)) {
-        url += "&hd=" + URLEncoder.encode(domain,
+      if (domains.length == 1) {
+        url += "&hd=" + URLEncoder.encode(domains[0],
             StandardCharsets.UTF_8.name());
+      } else if (domains.length > 1) {
+        url += "&hd=*";
```

Comment:
What is the spec saying on multiple hosted domain? Shouldn't the param be coma separated list of domains?